styles.css

```diff
@@ -136,6 +135,7 @@ i.fa-solid{
   justify-content:center;
   align-items:center;
   color:white;
+  cursor: pointer;
 }
 
 a[href*="github"]:link {
```

Comment:
Well done with using a complex selector! 
In real life you'd want to avoid using it unless it's completely unavoidable (E.g. there are cases when code comes from external library and element you want to change is deep inside another element. And classes are generated dynamically at runtime so you cannot use them so that you could rely on a selector like this).